packages/pool/contracts/TimelockUtils.sol

```diff
@@ -46,13 +46,15 @@ abstract contract TimelockUtils is ClaimUtils, ITimelockUtils {
             msg.sender == timelockManager,
             "Pool: Caller not TimelockManager"
             );
-        users[userAddress].unstaked += amount;
+        uint256 unstakedUpdate = users[userAddress].unstaked + amount;
+        users[userAddress].unstaked = unstakedUpdate;
         // Should never return false because the API3 token uses the
         // OpenZeppelin implementation
         assert(api3Token.transferFrom(source, address(this), amount));
         emit DepositedByTimelockManager(
```

Comment:
Should we also add source to the event just for completeness and consistency since we are adding all inputs on other events?

Reply:
`source` will always be `timelockManager` (which is `immutable`) due to the awkward way TimelockManager.sol was implemented, so not really needed (could be done for consistency but meh)